src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.csproj

```diff
@@ -104,19 +104,25 @@
       <Output TaskParameter="ResolvedDependencyFiles" ItemName="ResolvedDependencies" />
     </ResolveAssemblyReference>
 
+    <!-- Define the expected dependencies of the output assembly.
+         IMPORTANT: When modifying this list, coordinate with the MSBuild team to ensure:
+         1. Each dependency has a corresponding binding redirect in MSBuild.exe.
+         2. Dependency version fall within the ranges specified in MSBuild.exe binding redirects.
+         Note: Verify versions against the binding redirects of the minimum supported MSBuild version, located at: src/Layout/redist/minimumMSBuildVersion
+         You can find the MSBuild.exe binding redirects here: https://github.com/dotnet/msbuild/blob/<msbuild version branch>/src/MSBuild/app.amd64.config  -->
```

Comment:
So we've verified that these updated assembly versions are available in VS and part of the MSBuild binding redirects?

Reply:
Do you have a recommended way of testing that? My plan was to get this in now and try to test an insertion of RC2 ahead of the official one.